### ISSUE TITLE
Integration tests fail with a babylon syntax error

### ISSUE THREAD

**Issue by Profpatsch** [purescript-contrib/pulp#346]
```
$ node_modules/.bin/mocha test-js --require babel/register
/nix/store/42f6w07cbq0y5vjm3airici6926n29hz-babel-core-5.8.38/lib/transformation/file/index.js:671
      throw err;
      ^

SyntaxError: /nix/store/zbhyj887pkzcjdgjhf5p6cdijiq61vw5-temp-0.8.3/lib/temp.js: Invalid number (226:20)
function mkdir(affixes, callback) {
  var dirPath = generateName(affixes, 'd-');
  fs.mkdir(dirPath, 0700, function(err) {
    if (!err) {
      deleteDirOnExit(dirPath);
    }
    at Parser.pp.raise (/nix/store/pdyrgg4ky9cgzznq1jdxxkl01992fcqc-babylon-5.8.38/lib/parser/location.js:24:13)
```

I’m not exactly sure what all that stuff does, but [this looks like a relevant issue](https://github.com/prettier/prettier/issues/228). Octal numbers like `0700` (as opposed to `0o700`) lead to such errors.

**Comment by hdgarrood** [comment]
This should be fixed since I've updated package.json to require `temp` >= 0.9.0, as the octal literals have been removed.

**Comment by hdgarrood** [comment]
Fixed by 4157fcf27b7bb23c5b1366802456fd5271efa0ab
